src/table/expandable-rows/styles.scss

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+@use '../../internal/styles/index' as styles;
+@use '../../internal/styles/tokens' as awsui;
+
+.expand-toggle-hidden.expand-toggle-hidden {
```

Comment:
Why two times the same class name for the same element?

Reply:
That is needed for specificity. The same class is used below to assign other styles and it takes precedence. An alternative could be to assign both `expand-toggle` and `expand-toggle-hidden`. However, the `expand-toggle` is also used in test-utils and it must not match the hidden toggles.

Reply:
Can't the test utils exclude the hidden class instead? Something like

```
this.findNativeTable().find(`tbody tr:nth-child(${rowIndex}) .${expandableRowsStyles['expand-toggle']}:not(${expandableRowsStyles['expand-toggle-hidden']})`)
```

Reply:
Updated!